setup.py

```diff
@@ -84,24 +84,25 @@ def merge_req_lists(*req_lists: List[str]) -> List[str]:
 ]
 
 otl_dependencies = [
-    "opentelemetry-api==1.17.0",
-    "opentelemetry-exporter-otlp==1.17.0",
-    "opentelemetry-exporter-otlp-proto-grpc==1.17.0",
-    "opentelemetry-exporter-otlp-proto-http==1.17.0",
-    "opentelemetry-instrumentation==0.38b0",
-    "opentelemetry-proto==1.17.0",
-    "opentelemetry-sdk==1.17.0",
-    "opentelemetry-semantic-conventions==0.38b0",
+    "opentelemetry-api>=1.20.0",  # latest api required
+    "opentelemetry-exporter-otlp>=1.20.0",  # latest api required
+    "opentelemetry-exporter-otlp-proto-grpc>=1.20.0",  # latest api required
+    "opentelemetry-exporter-otlp-proto-http>=1.20.0",  # latest api required
+    "opentelemetry-instrumentation>=0.38b0",
+    "opentelemetry-proto>=1.20.0",  # latest api required
+    "opentelemetry-sdk>=1.20.0",  # latest api required
+    "opentelemetry-semantic-conventions>=0.41b0",
 ]
 
 stats_dependencies = merge_req_lists(
     _sql_dependencies,
-    requests_requirements,
     otl_dependencies,
     [
-        "wrapt==1.15.0",
-        "tqdm==4.62.3",
-        "omegaconf>=2.2.2",
+        "protobuf==3.20.3",  # required by ydb & otlp
+        "requests",
+        "wrapt",
+        "tqdm",
+        "omegaconf>=2.2.2",  # latest api required
```

Comment:
Same, could you specify which aspects of the new version are needed.

Reply:
Updated